pyproject.toml

```diff
@@ -41,6 +100,12 @@ doctest_optionflags = "NORMALIZE_WHITESPACE ELLIPSIS NUMBER"
 minversion = "6.0"
 testpaths = "cf_units"
 
+[tool.setuptools]
+license-files = ["LICENSE"]
```

Comment:
I don't think this is necessary https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#controlling-files-in-the-distribution

Reply:
Good spot, I have tried this in f4ee03a0b9642039564b4147fe396965e5141411 and that shows this working:

https://test.pypi.org/project/cf-units/3.2.1.dev89/

>License: BSD License (BSD 3-Clause License Copyright (c) 2015, Met Office. All rights reserved. Redistribution and use i...)